app/models/container_node.rb

```diff
@@ -5,6 +5,8 @@ class ContainerNode < ApplicationRecord
   include NewWithTypeStiMixin
   include TenantIdentityMixin
   include SupportsFeatureMixin
+  include ArchivedMixin
+  include_concern 'Purging'
```

Comment:
Best remove (that's what we did with all the other classes using `ArchivedMixin`)
```
delegate :my_zone, :to => :ext_management_system
```